lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -33,8 +33,8 @@ class ExampleWording < Cop
         MSG_SHOULD = 'Do not use should when describing your tests.'.freeze
         MSG_IT     = "Do not repeat 'it' when describing your tests.".freeze
 
-        SHOULD_PREFIX = 'should'.freeze
-        IT_PREFIX     = 'it '.freeze
+        SHOULD_PREFIX = /\Ashould(?:n't|not)?\b/i
```

Comment:
> There seems to have been a race condition in our agreement 😉

haha!

When I ran `mutest`, it actually caught this case. But I forgot to inline this particular regex (i did all the ones in the `Wording` class). I need to add static constant mutation support to `mutest` one of these days...